modules/RedisTimeSeriesEdge/Program.cs

```diff
@@ -16,6 +18,7 @@ namespace RedisTimeSeriesEdge
     class Program
     {
         const string RedisHostNameKey = "REDIS_HOST_NAME";
+        const int MetricsPort = 9121;

```

Comment:
I hardcoded the value for simplicity, we just need to pick any port number. But it also can be passed as env var. And then I'll need to update dockerfile and module.json. Let me know if you prefer this option.